modules/contact/server-ts/resolvers.ts

```diff
@@ -28,10 +32,8 @@ export default () => ({
         });
       } catch (e) {
         log.error(e);
-        return { errors: [{ field: 'serverError', message: t('contact:sendError') }] };
+        throw new ApolloError(t('contact:sendError'), 'CONTACT_DENIED');
```

Comment:
Why did you choose `ApolloError` class while the library provides more specific `UserInputError`? (https://www.apollographql.com/docs/apollo-server/features/errors.html#Augmenting-error-details)